pages/common/clamscan.md

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
Same here https://docs.clamav.net/manual/Usage/Scanning.html#clamscan

Reply:
`clamscan` can have just one `More information` link. I don't understand why there are two suggestions for this page.

[first suggested fix](https://github.com/tldr-pages/tldr/pull/9578#discussion_r1038133995)